cypress/integration/sample_spec.js

```diff
@@ -38,4 +39,265 @@ describe('Task', () => {
+       cy.contains('First Name').should('exist').click()
+       cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(1)")
+
+       .then(tableBody => {
+        let sortNamefromSite = [];
+        sortNamefromSite.push(tableBody.text());
+        cy.log(sortNamefromSite)
+        let sortNameHere = [];
+        sortNameHere.push(tableBody.text())
+        cy.log(sortNameHere)
+        expect(sortNameHere.sort()).to.eql(sortNamefromSite);
+    })
+        cy.contains('Last Name').should('exist').click()
+        cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(2)")
+
+        .then(tableBody2 => {
+        let sortLastfromSite = [];
+        sortLastfromSite.push(tableBody2.text());
+        cy.log(sortLastfromSite)
+        let sortLastNameHere = [];
+        sortLastNameHere.push(tableBody2.text())
+        cy.log(sortLastNameHere)
+        expect(sortLastNameHere.sort()).to.eql(sortLastfromSite)
+    })    
+    
+
+        cy.contains('Age').should('exist').click()
+        cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(3)")
+
+        .then(tableBody3 => {
+        let sortAgefromSite = [];
+        sortAgefromSite.push(tableBody3.text());
+        cy.log(sortAgefromSite)
+        let sortAgeHere = [];
+        sortAgeHere.push(tableBody3.text())
+        cy.log(sortAgeHere)
+        expect(sortAgeHere.sort()).to.eql(sortAgefromSite)
+   
+     }) 
+
+        cy.contains('Email').should('exist').click()
+        cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(4)")
+
+        .then(tableBody4 => {
+        let sortEmailfromSite = [];
+        sortEmailfromSite.push(tableBody4.text());
+        cy.log(sortEmailfromSite)
+        let sortEmailHere = [];
+        sortEmailHere.push(tableBody4.text())
+        cy.log(sortEmailHere)
+        expect(sortEmailHere.sort()).to.eql(sortEmailfromSite)
+
+
+    }) 
+        cy.contains('Salary').should('exist').click()
+        cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(5)")
+
+        .then(tableBody5 => {
+        let sortSalaryfromSite = [];
+        sortSalaryfromSite.push(tableBody5.text());
+        cy.log(sortSalaryfromSite)
+        let sortSalaryHere = [];
+        sortSalaryHere.push(tableBody5.text())
+        cy.log(sortSalaryHere)
+        expect(sortSalaryHere.sort()).to.eql(sortSalaryfromSite)
+
+
+    }) 
+        cy.contains('Department').should('exist').click()
+        cy.get(".rt-tbody > [role='rowgroup'] > [role='row']").find("div:nth-child(6)")
+
+        .then(tableBody6 => {
+        let sortDepartmentfromSite = [];
+        sortDepartmentfromSite.push(tableBody6.text());
+        cy.log(sortDepartmentfromSite)
+        let sortDepartmentHere = [];
+        sortDepartmentHere.push(tableBody6.text())
+        cy.log(sortDepartmentHere)
+        expect(sortDepartmentHere.sort()).to.eql(sortDepartmentfromSite)
+
+
+    }) 
+
+      
+}) 
+     
```

Comment:
U are able to create a few reusable functions, u have a lot of copy pasted code

Reply:
can you share with me some links how to do that?